### PR TITLE
Make better universal 404 page [resolves #46]

### LINKED ISSUE #46: Make better universal 404 page
-Inspiration- fail whale

### PR DESCRIPTION
**Pull request by Anasauce**

### CODE REVIEW

File: browser/actions/startPolingForPrrrs.js [hunk 1]
```diff
@@ -24,7 +24,8 @@ const scheduleNextLoad = function(){
   timeout = setTimeout(pullForPrrs, POLLING_DELAY)
 }
 
-pullForPrrs()
+// pullForPrrs()
```
Comment (deadlyicon) [inline browser/actions/startPolingForPrrrs.js]: whopoooopspspsss

File: browser/components/pages/NotFoundPage.js [hunk 2]
```diff
@@ -1,12 +1,47 @@
...
+    else if ( num === 1 ) {
+      return sadCat2
+    }
+    else if ( num === 2 ) {
+      return sadCat3
+    }
+    else {
+      return sadCat4
+    }
+  }
```
Comment (deadlyicon) [inline browser/components/pages/NotFoundPage.js]: This shows me you need to work on your algorithms. Please take a shot at significantly simplifying this method.

File: browser/components/pages/NotFoundPage.js [hunk 3]
```diff
@@ -1,12 +1,47 @@
...
+
   render(){
+    // console.log("hmmmm", this.whichSadCat())
```
Comment (deadlyicon) [inline browser/components/pages/NotFoundPage.js]: please remove junk comments

File: browser/components/pages/NotFoundPage.js [hunk 4]
```diff
@@ -1,12 +1,47 @@
 import React, { Component } from 'react'
+import sadCat1 from '../../images/sadcat1.gif'
+import sadCat2 from '../../images/sadcat2.gif'
+import sadCat3 from '../../images/sadcat3.gif'
+import sadCat4 from '../../images/sadcat4.gif'
```
Comment (deadlyicon) [inline browser/components/pages/NotFoundPage.js]: hahaha awesome!!!

File: browser/components/pages/NotFoundPage.js (hunk 2 above)
Comment (Anasauce) [inline browser/components/pages/NotFoundPage.js]: sorry yeah, we were running out of time...we can refactor over the weekend

Comment (deadlyicon) [inline browser/components/pages/NotFoundPage.js]: MERGED! please fix later

**Pull request merged**